Sources/Memcache/ChannelHandler/MemcacheDecodingError.swift

```diff
@@ -0,0 +1,52 @@
+import NIOCore
+
+struct MemcacheDecodingError: Error {
+    let messageVerb: String
+    let payload: String
+    let description: String
+    let file: StaticString
+    let line: UInt
+
+    static func withPartialError(
+        _ partialError: MemcachePartialDecodingError,
+        messageVerb: String,
+        messageBytes: ByteBuffer
+    ) -> Self {
+        MemcacheDecodingError(
+            messageVerb: messageVerb,
+            payload: "", // TODO: Can we get a base64 representation without Foundation?
+            description: partialError.description,
+            file: partialError.file,
+            line: partialError.line
+        )
+    }
+
+    static func emptyMessageReceived(
+        bytes: ByteBuffer,
+        file: StaticString = #file,
```

Comment:
I think we should use `file: String = #fileID` here. `StaticString` and `#file` are out of fashion for new code.